rfc6962-bis.xml

```diff
@@ -1203,6 +1203,14 @@ entry specified by <spanx style="verb">start</spanx>.
         </t>
       </section>
     </section>
+    <section title="TLS Feature Extension">
+      <t>
+	A CA MAY include the CT TLS extension number in a <xref target="RFC7633">TLS Feature Extension</xref> in root or intermediate certificates. This indicates that CT compliance is required for all certificates which are directly or indirectly issued by such certificates.
+      </t>
+      <t>
+	Clients MUST treat certificates which fail to comply with this requirement as invalid.
```

Comment:
Good point.  A monitor could spot failures to comply with this requirement, for example.